examples/joyid/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "private": true,
+  "name": "@lumos-examples/joyid",
+  "version": "0.21.0-next.0",
```

Comment:
Version is different from https://github.com/ckb-js/lumos/pull/659/files#diff-497d915e1ad4f79606d1e44cf2b5c40333d4ef1aa2b1b1fda192f1e9e3475655R3